userguide/drawing_classifier/data-preparation.md

```diff
@@ -18,8 +18,8 @@ both as bitmaps and as stroke-based drawings.
 Open a new terminal window and run the following commands:
 
 ```
-$ mkdir -p ~/Downloads/quickdraw
-$ cd ~/Downloads/quickdraw
+$ mkdir -p /Users/$(whoami)/Downloads/quickdraw
```

Comment:
I don't think this generalizes to other platforms

Reply:
Doesn't `whoami` work on both macOS and Linux?

Reply:
Is it safe to assume that we should write to /Users?

Could we just use the current working directory here, for maximum generality?

Reply:
Done.